common/viewer.cpp

```diff
@@ -1930,8 +1941,8 @@ namespace rs2
 
         auto bottom_y = win.framebuf_height() - viewer_rect.y - viewer_rect.h;
         
-        glViewport(static_cast<GLint>(viewer_rect.x), static_cast<GLint>(bottom_y),
-            static_cast<GLsizei>(viewer_rect.w), static_cast<GLsizei>(viewer_rect.h - top_bar_height));
+        glViewport(static_cast<GLint>(FORCE_NON_NEGATIVE(viewer_rect.x)), static_cast<GLint>(FORCE_NON_NEGATIVE(bottom_y)),
```

Comment:
the above refactor may save the explicit casts and make it more readable

Reply:
Still need the explicit cast , otherwise we get a warning